tests/models/mbart/test_modeling_flax_mbart.py

```diff
@@ -462,3 +470,153 @@ def translate_src_text(self, **tokenizer_kwargs):
     @slow
     def test_batch_generation_en_ro(self):
         self._assert_generated_batch_equal_expected()
+
+
+class MBartStandaloneDecoderModelTester:
```

Comment:
Is this identical to `FlaxBartStandaloneDecoderModelTester`? https://github.com/huggingface/transformers/blob/c3a93d8d821bc1df3601ba858e7385eada8db3a5/tests/models/bart/test_modeling_flax_bart.py#L675
We can copy one-to-one from this testing code